src/modals/confirm-sign-modal/use-sign-message.hook.ts

```diff
@@ -73,7 +72,7 @@ export const useSignMessage = (formValues: CreateNftFormValues) => {
             )
           ),
           catchError(e => {
-            showErrorToast({ description: getAxiosQueryErrorMessage(e) });
+            showErrorToast({ description: e.message, isCopyButtonVisible: true });
```

Comment:
I meant something like this:
```suggestion
            let description = getAxiosQueryErrorMessage(e);
            let contentToCopy = getAxiosQueryErrorMessage(e);
            if (e instanceof AxiosError) {
              const responseStatus = e.response?.status;
              const responseData = e.response?.data;
              const responseMessage = responseData?.message ?? responseData?.error;
              if (responseStatus === 500) {
                contentToCopy = responseMessage;
              } else if (responseStatus >= 400 && responseStatus <= 499) {
                description = responseMessage;
                contentToCopy = responseMessage;
              }
            }
            showErrorToast({ description, isCopyButtonVisible: true, onPress: () => copyStringToClipboard(contentToCopy) });
```
However, it should be moved into a separate function because it is used in one more place